cloudify_cli/config/parser_config.py

```diff
@@ -244,26 +246,27 @@ def auto_generate_ids_argument():
 
 def parser_config():
     return {
-        'description': 'Manages Cloudify in different Cloud Environments',
+        'description': 'A Command Line Interface Cloudify',
```

Comment:
This change is great in concept. 
But what about something in the likes of "Cloudify's Command Line Interface"? I'm not familiar with the sentence structuring you proposed.